dist/index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>CEDICT Server Page</title>
+  <script src="service.js"></script>
+</head>
+<body>
+<p>This is a CEDICT server page.</p>
+</body>
+</html>
```

Comment:
A very simple page whose sole purpose is to load itself within an iframe and, by that, pull the `service.js` script into an iframe.